src/components/shared/footer.tsx

```diff
@@ -82,6 +83,23 @@ interface FooterCcpaProps {
     isCcpa: boolean;
 }
 
-const FooterCcpa: FC<FooterCcpaProps> = ({ isCcpa }) => <Footer>{footerContents(isCcpa)}</Footer>
+const renderContent = (ccpaStatus: boolean): JSX.Element => {
+    if (!ccpaStatus){
+        return <a css={anchor} href="https://www.theguardian.com/help/privacy-policy">Privacy Settings</a>
+    } else {
+        return <a css={anchor} href="https://www.theguardian.com/help/privacy-policy">California Residents - Do Not Sell</a>
```

Comment:
What does the "Do Not Sell" mean?

Reply:
Not quite sure :) it was on the design system text content

Reply:
I believe it's specific wording related to CCPA - perhaps @petternitter can confirm?

Reply:
Yes, it's the California law thing. I raised the fact that users might not know what this is about with @petternitter . It'd be good to review the copy at some point to make it more intuitive.